### ISSUE TITLE
завершение рисования трека

### ISSUE THREAD

**Issue by slazav** [wladich/nakarte#1189]
Небольшая неаккуратность в интерфейсе, которая иногда раздражает:

Рисуем трек, нажимаем ESC, рисование завершается.

Рисуем трек, но перед завершением тыкаем в одно их меню: переключаем слой, например. После этого ESC перестает работать. Приходится завершать трек каким-то нестандартным способом: ставя и удаляя лишнюю точку или через меню.

Хочется, чтобы завершение трека по ESC работало всегда.

**Comment by myadzel** [comment]
Тоже этот баг мешает.

Если кликнуть на шестеренку или свернуть слой, то все нормально потом работает по Esc.

Нашёл проблему: не возвращается фокус на карту при клике по слоям, проблема в Leaflet.

Решение: возвращать фокус при нажатии Esc.

**Comment by wladich** [comment]
Нет, проблема не в leaflet, а  в https://github.com/wladich/nakarte/blob/7bda8342e49a07762cf8dae5cddc067409935b4f/src/lib/leaflet.polyline-edit/index.js#L166
